masim.c

```diff
@@ -56,7 +56,7 @@ void pr_phase(struct phase *phase)
 	struct access *pattern;
 	int j;
 
-	printf("Phase (%s) for %u ms\n", phase->name, phase->time_ms);
+	printf("Phase (%s) for %lu ms\n", phase->name, phase->time_ms);
```

Comment:
This change is for addresses, not time, right?  I think this change is not really necessary?